app/controllers/application_controller.rb

```diff
@@ -120,10 +120,14 @@ def skip_sentry_exception_types
     end
 
     headers['WWW-Authenticate'] = 'Token realm="Application"' if va_exception.is_a?(Common::Exceptions::Unauthorized)
-    render json: { errors: va_exception.errors }, status: va_exception.status_code
+    render_errors(va_exception)
   end
   # rubocop:enable Metrics/BlockLength
 
+  def render_errors(va_exception)
+    render json: { errors: va_exception.errors }, status: va_exception.status_code
+  end
```

Comment:
Sorry -- makes sense after you see the rest :)

Reply:
Yup, there's actually elsewhere in the code where something similar is done with `render_job_id.` 

The inheritance chain is not immediately clear, because no changes were needed to VAOS::V0:BaseController. That controller, inherits from ApplicationController, but it's the VAOS::V1:BaseController that needs to serialize errors a bit differently.